src/lib/CombatCalc.ts

```diff
@@ -1183,6 +1183,9 @@ export default class CombatCalc {
       // https://twitter.com/JagexAsh/status/1133350436554121216
       dist = dist.transform(divisionTransformer(3));
     }
+    if (this.monster.name === 'Slagilith' && this.player.equipment.weapon?.category !== EquipmentCategory.PICKAXE) {
```

Comment:
Maybe add the twitter link as a comment. https://twitter.com/JagexAsh/status/1219652159148646401